Data/SourceAssets/Localization/English/EndMsg.txt

```diff
@@ -1,15 +1,15 @@
-DESPERABIS ist freeware, d.h. die Nutzung und Verbreitung ist erlaubt.
-Wenn Dir das Spiel gef�llt, bitte ich um email-Nachricht an:
+DESPERABIS is freeware, and thus, free use and distribution is permitted.

```

Comment:
fyi, none of the links (or email) still works, so I'm considering to re-link to github. But we can do this later.

Reply:
noted